src/issuer/pre-signup/user/app.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from aws_lambda_powertools.logging import correlation_paths
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools import Tracer
+from aws_lambda_powertools import Metrics
+from aws_lambda_powertools.metrics import MetricUnit
+from pyqldb.driver.qldb_driver import QldbDriver
+from pyqldb.config.retry_config import RetryConfig
+from qldb_helper import QLDBHelper
+
+tracer = Tracer()
+logger = Logger()
+metrics = Metrics(namespace="Powertools")
+
+
+@tracer.capture_method
+def signup_confirmation(event: dict, context: LambdaContext):
+    # adding custom metrics
+    # See: https://awslabs.github.io/aws-lambda-powertools-python/latest/core/metrics/
+    metrics.add_metric(name="SignupConfirmationInvocations", unit=MetricUnit.Count, value=1)
+
+    # structured log
+    # See: https://awslabs.github.io/aws-lambda-powertools-python/latest/core/logger/
+    logger.info("LedgerStore API - issuer/pre-signup/user HTTP 200")
+    logger.info("request: {request}", request=event.get("request"))
+    logger.info("event: {event}", event=event)
+    user_sub = event.get("userName")
+    retry_config = RetryConfig(retry_limit=3)
```

Comment:
Is this retry config shared in multiple places? Could go with a global there, same applies for other shared variables

Reply:
good call! yeah it's used everywhere. Will put this in somewhere reusable